ras_party/controllers/account_controller.py

```diff
@@ -65,11 +65,11 @@ def post_respondent(party, tran, session):
 
     iac = request_iac(party['enrolmentCode'])
     if not iac.get('active'):
-        raise RasError("Enrolment code is not active.", status=400)
+        raise ClientError("Enrolment code is not active.", status=400)
```

Comment:
👍

Reply:
The error messages passed to `ClientError` aren't used - they don't appear in the logs.